sktime/forecasting/ets.py

```diff
@@ -0,0 +1,338 @@
+    initial_trend : float, optional
+        The initial trend component. Only used if initialization is 'known'.
+    initial_seasonal : array_like, optional
+        The initial seasonal component. An array of length `seasonal_periods`.
+        Only used if initialization is 'known'.
+    bounds : dict or None, optional
+        A dictionary with parameter names as keys and the respective bounds
+        intervals as values (lists/tuples/arrays).
+        The available parameter names are, depending on the model and
+        initialization method:
+
+        * "smoothing_level"
+        * "smoothing_trend"
+        * "smoothing_seasonal"
+        * "damping_trend"
+        * "initial_level"
+        * "initial_trend"
+        * "initial_seasonal.0", ..., "initial_seasonal.<m-1>"
+
+        The default option is ``None``, in which case the traditional
+        (nonlinear) bounds as described in [1]_ are used.
+    start_params : array_like, optional
+        Initial values for parameters that will be optimized. If this is
+        ``None``, default values will be used.
+        The length of this depends on the chosen model. This should contain
+        the parameters in the following order, skipping parameters that do
+        not exist in the chosen model.
+
+        * `smoothing_level` (alpha)
+        * `smoothing_trend` (beta)
+        * `smoothing_seasonal` (gamma)
+        * `damping_trend` (phi)
+
+        If ``initialization_method`` was set to ``'estimated'`` (the
+        default), additionally, the parameters
+
+        * `initial_level` (:math:`l_{-1}`)
+        * `initial_trend` (:math:`l_{-1}`)
+        * `initial_seasonal.0` (:math:`s_{-1}`)
+        * ...
+        * `initial_seasonal.<m-1>` (:math:`s_{-m}`)
+
+        also have to be specified.
+    maxiter : int, optional
+        The maximum number of iterations to perform.
+    full_output : bool, optional
+        Set to True to have all available output in the Results object's
+        mle_retvals attribute. The output is dependent on the solver.
+        See LikelihoodModelResults notes section for more information.
+    disp : bool, optional
+        Set to True to print convergence messages.
+    callback : callable callback(xk), optional
+        Called after each iteration, as callback(xk), where xk is the
+        current parameter vector.
+    return_params : bool, optional
+        Whether or not to return only the array of maximizing parameters.
+        Default is False.
+    References
+    ----------
+    [1] Hyndman, R.J., & Athanasopoulos, G. (2019) *Forecasting:
+        principles and practice*, 3rd edition, OTexts: Melbourne,
+        Australia. OTexts.com/fpp3. Accessed on April 19th 2020.
+
+    The following parameters are adapted from the ones on R
+    (https://www.rdocumentation.org/packages/forecast/versions/8.12/topics/ets),
+    and are used for automatic model selection:
+
```

Comment:
I would move the section `References` to the end of the docstring, I see that you want to reference the different parameters to the different implementations, but I think it's okay to just mention in the references that they come from both model implementations